src/rust/ide/src/controller/visualization.rs

```diff
@@ -137,7 +137,7 @@ impl Handle {
                 let js_code    = self.language_server_rpc.read_file(&path).await?.contents;
                 let identifier = visualization.clone();
                 let error      = |_| VisualizationError::InstantiationError {identifier}.into();
-                let module     = data::builtin_library(); // FIXME: provide real library name.
+                let module     = enso::builtin_library(); // FIXME: provide real library name.
```

Comment:
is this fixme still valid?

Reply:
Yes, it is. Actually the code is funny: we set here a wrong name, but then in integration.rs we don't care about that field, but just set the Project.Main. I'll extend the FIXME and add the issue number.